src/main/java/Group15/Model/Workout.java

```diff
@@ -51,12 +50,26 @@ public String getName() {
         return name;
     }
 
-    public void setDescription(String description) {
-        this.description = description;
+    public String makeDescription() {
+        StringBuilder description = new StringBuilder();
+        int duration = calculateWorkoutDuration() / 60000;
+        description.append(duration + " min. ");
+        for (BodyPart bodyPart : usedBodyParts()) {
+            description.append(bodyPart.toString() + ", ");
```

Comment:
Could we remove the last comma. Just check if the description length is greater than zero or not empty and then remove 2 from the length.